josemak25/hairx-client
# Ft client signup screen3

#### What does this PR do?

Shows page 3 of 4 of signup, shows a screen where a client can create a password

#### Description of Task to be completed?

Have the create password screen show the following:

1. icon and pagination.
2. welcome text.
3. Input fields for client's full name, create a password
![IMG_2534](https://user-images.githubusercontent.com/51112934/80000426-08cfbb00-84b5-11ea-8d7b-2307fbc413cf.PNG)
 and confirm password.

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test
![IMG_2534](https://user-images.githubusercontent.com/51112934/80000541-2c930100-84b5-11ea-8412-974a96d2ab18.PNG)

Using an emulator: Launch the app, and you should see the home screen after loading.

#### Screenshots (if appropriate)
